Reference docs: what happens when you change the Fleet web address (#30999)
fleetdm/fleet: docs/REST API/rest-api.md

| ai_features_disabled              | boolean | Whether AI features are disabled.                                                           |
| query_report_cap                  | integer | The maximum number of results to store per query report before the report is clipped. If increasing this cap, we recommend enabling reports for one query at time and monitoring your infrastructure. (Default: `1000`) |

> > Note: If `server_url` changes hosts already are already enrolled will no longer communicate with Fleet. They need to be re-enrolled.

Author: ```suggestion
> Note: If `server_url` changes hosts already are already enrolled will no longer communicate with Fleet. They need to be re-enrolled. Also, if you're using Android MDM, you have to turn Android MDM on and back off to point Google to the new `server_url`.
```

> > Note: If `server_url` changes hosts already are already enrolled will no longer communicate with Fleet. They need to be re-enrolled. Also, if you're using Android MDM, you have to turn Android MDM on and back off to point Google to the new `server_url`.

Reviewer: ```suggestion
> Note: If `server_url` changes hosts already are already enrolled will no longer communicate with Fleet. They need to be re-enrolled. Also, if you're using Android MDM, you have to turn Android MDM off and back on to point Google to the new `server_url`.
```

> > Note: If `server_url` changes hosts already are already enrolled will no longer communicate with Fleet. They need to be re-enrolled. Also, if you're using Android MDM, you have to turn Android MDM off and back on to point Google to the new `server_url`.

Author: ```suggestion
> Note: If `server_url` changes hosts already are already enrolled will no longer communicate with Fleet. They need to be re-enrolled. If you're using Android MDM, before to re-enroll Android hosts, you have to turn Android MDM off and back on to point Google to the new `server_url`.
```
